Marlin/Configuration.h

```diff
@@ -884,6 +884,7 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 // leaving it undefined or defining as 0 will disable the servo subsystem
 // If unsure, leave commented / disabled
 //
+//#define BLTOUCH_MENU // uncomment if you want the reset BLTouch menu item
```

Comment:
Not the best place for this. We don't have a "probes and endstops" section, _per se_, but we should try to go in that general direction. So, after endstops, then servos, then probes… something like that…. But not on this line, especially as it comes immeditely after the comment describing `NUM_SERVOS`.